### PR TITLE
Small moves: part two

### PR DESCRIPTION
**Pull request by keirog**
General tidy of the meta bits ✌️

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
   "main": "proxy.js",
   "scripts": {
-    "precommit": "secret-squirrel",
     "commitmsg": "secret-squirrel-commitmsg",
-    "test": "make test",
-    "test-debug": "make test-debug"
+    "precommit": "secret-squirrel",
+    "start": "node proxy.js",
+    "test": "mocha -R spec tests/routing.js",
+    "test-debug": "mocha --debug-brk -R spec tests/routing.js"
```
Comment (AbdullahAli) [inline package.json]: Can we change this to `test:debug` instead please

File: package.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
 {
   "name": "ft-email-router",
-  "private": "true",
-  "version": "0.0.1",
+  "private": true,
+  "version": "0.0.0",
```
Comment (AbdullahAli) [inline package.json]: Is this change intentional? `0.0.0`

File: Makefile [hunk 3]
```diff
@@ -1,14 +0,0 @@
...
-	mocha --debug-brk -R spec tests/routing.js
-
-run:
-	export DYNO='run.2665'; node proxy.js
-
-install:
-	npm install
-
-clean:
-	git clean -fxd
```
Comment (AbdullahAli) [inline Makefile]: 🙌

File: package.json (hunk 2 above)
Comment (keirog) [reply package.json]: Yep, just to (hopefully) imply that it’s not a versioned package

File: Makefile [hunk 4]
```diff
@@ -1,14 +0,0 @@
-test:
-	mocha -R spec tests/routing.js
-
-test-debug:
-	mocha --debug-brk -R spec tests/routing.js
-
-run:
-	export DYNO='run.2665'; node proxy.js
```
Comment (AbdullahAli) [inline Makefile]: Do we still need to expose this env var in heroku?

File: package.json (hunk 2 above)
Comment (AbdullahAli) [reply package.json]: 👍🏼

File: Makefile (hunk 4 above)
Comment (keirog) [reply Makefile]: Heroku never used this command (it was running `node proxy.js` from the Procfile)
